src/app/features/features.module.ts

```diff
@@ -21,13 +24,16 @@ import { ProfileModule } from './profile/profile.module';
         UsersPageComponent
     ],
     imports: [
+        BrowserAnimationsModule,
+        CommonModule,
+        EffectsModule.forFeature([UsersEffects]),
+        HttpClientModule,
         LayoutModule,
+        MatIconModule,
+        MatSnackBarModule,
```

Comment:
I am not sure but maybe you don't need to add `MatSnackBarModule` here since it is operated through a service and provided globally
try to remove and check if it is working

Reply:
Don't work without `MatSnackBarModule`

Reply:
What if you will try to import it in NotificationsModule instead?
If won't work - in app.module